Mlem/Extensions/View - Handle Lemmy Links.swift

```diff
@@ -188,7 +188,7 @@ struct HandleLemmyLinkResolution: ViewModifier {
                             // so I'll skip and let the system open it instead
                         // }
                     } catch {
-                        print(String(describing: error))
+                        appState.contextualError = .init(underlyingError: error)
```

Comment:
an example of using the new `.contextualError` where we don't want to communicate anything to the user, so we only pass in the error itself, the new handling will look at that error and if it's a session issue, or general API error we will display it